addons/mail/static/src/core/common/composer_actions.js

```diff
@@ -132,6 +132,18 @@ composerActionsRegistry
         name: _t("Open Full Composer"),
         onClick: (component) => component.onClickFullComposer(),
         sequence: 30,
+    })
+    .add("add-canned-response", {
+        condition: (component) =>
+            component.store.hasCannedResponses &&
+            component.thread &&
+            component.env.services["mail.suggestion"]
+                .getSupportedDelimiters(component.thread)
+                .find(([delimiter]) => delimiter === "::"),
+        icon: "fa fa-file-text-o",
+        name: _t("Insert a Canned response"),
```

Comment:
Shouldn't we update the translation file?

Reply:
Indeed, I'll do it